server/plugin/command.go

```diff
@@ -268,7 +268,9 @@ func azureDevopsDisconnectCommand(p *Plugin, c *plugin.Context, commandArgs *mod
 }
 
 func executeDefault(p *Plugin, c *plugin.Context, commandArgs *model.CommandArgs, args ...string) (*model.CommandResponse, *model.AppError) {
-	return p.sendEphemeralPostForCommand(commandArgs, constants.InvalidCommand)
+	out := constants.InvalidCommand + "\n\n" + constants.HelpText
```

Comment:
here you can add the "\n\n" with the invalid command constant as now you are not going to use this alone